### DISCUSSION TITLE
Disable all errors on Cefsharp WinForms

### DISCUSSION THREAD

**Discussion by NedimBadzak** [cefsharp/CefSharp#4436] [General]
Let's say I have this following proof of concept webpage:
```
<html>

  <body>
    <h1>Hello</h1>
  </body>


<script>
console.log("hahah")
z
</script>
</html>
```
If I open this page with CefSharp browser, I get the following message in the bottom left corner of the screen:
![image](https://user-images.githubusercontent.com/23142003/227745321-74de8415-9057-44c4-8c74-70ae24b8c9c8.png)

Now, this would be fine and helpful in development use, but in production use, I'd like to disable **all** errors visually.

I tried overriding the `IJsDialogHandler` and the OnJSDialog method like this:
```
        public bool OnJSDialog(IWebBrowser chromiumWebBrowser, IBrowser browser, string originUrl, CefJsDialogType dialogType, string messageText, string defaultPromptText, IJsDialogCallback callback, ref bool suppressMessage)
        {
            suppressMessage = true;
            return false;
        }
```

But I still get the same error messages. 

So the question is, how would I go about overriding the behavior and disabling ALL JS error messages?

Thanks in advance!

**Comment by amaitland** [comment]
> If I open this page with CefSharp browser, I get the following message in the bottom left corner of the screen:

`ConsoleMessages` aren't displayed by default, it's up to the embedding application to actually show them. 

The image you've shown looks a lot like the `MinimalExample` which shows the messages for demo purposes.
